stage2/05-docker-compose/01-run.sh

```diff
@@ -6,7 +6,7 @@ echo "Installing docker-compose from pip3, and also setting up the box folder st
 on_chroot << EOF
 pip3 install docker-compose
 cd /home/${FIRST_USER_NAME}
-wget -qO- "https://raw.githubusercontent.com/getumbrel/umbrel-compose/master/install-box.sh" | sh
+wget -qO- "https://raw.githubusercontent.com/mayankchhabra/umbrel-compose/feature-ota/install-box.sh" | sh
```

Comment:
Change to `getumbrel/umbrel-compose` repo once `feature-ota` branch of mayankchhabra/umbrel-compose is merged